advanced/editor-command-identifiers.md

```diff
@@ -7,93 +7,465 @@ description: The complete list of exposed editor commands.
+| italic         | Toggles italic on the selected content.                                                                                             |
+| underline      | Toggles an underline on the selected content.                                                                                       |
+| strikethrough  | Toggles strikethrough on the selected content.                                                                                      |
+| blockquote     | Toggles a blockquote on the selected content.                                                                                       |
+| subscript      | Toggles subscript on the selected content.                                                                                          |
+| superscript    | Toggles superscripts on the selected content.                                                                                       |
+| code           | Toggles a code (`<code>`) block on the selected content.                                                                            |
+| p              | Toggles a paragraph (`<p>`) block on the selected content.                                                                          |
+| h1             | Toggles a level 1 heading (`<h1>`) block on the selected content.                                                                   |
+| h2             | Toggles a level 2 heading (`<h2>`) block on the selected content.                                                                   |
+| h3             | Toggles a level 3 heading (`<h3>`) block on the selected content.                                                                   |
+| h4             | Toggles a level 4 heading (`<h4>`) block on the selected content.                                                                   |
+| h5             | Toggles a level 5 heading (`<h5>`) block on the selected content.                                                                   |
+| h6             | Toggles a level 6 heading (`<h6>`) block on the selected content.                                                                   |
+| div            | Toggles a division (`div>`) block on the selected content.                                                                          |
+| address        | Toggles an address (`<address>`) block on the selected content.                                                                     |
+| pre            | Toggles a preformated (`<pre>`) block on the selected content.                                                                      |
+| samp           | Toggles a sample (`<samp>`) block on the selected content.                                                                          |
+| forecolor      | Toggles a highlight color on the selected content. For example: `execCommand('mceToggleFormat', 'forecolor', '#FF0000')`            |
```

Comment:
Just so it doesn't get lost, I realized I gave the wrong info here sorry :man_facepalming: These 5 options can't be used with `FormatBlock` or `mceToggleFormat` sorry.

Reply:
noted